backend/crud.py

```
@@ -314,3 +315,184 @@ def delete_spot(db: Session, spot_id: UUID, user_id: UUID) -> None:
+
+
+def submit_quest_report(
+    db: Session,
+    quest_id: UUID,
+    walker_id: UUID,
+    payload: schemas.QuestReportPayload,
+) -> models.Quest:
+    """ウォーカーの報告を登録しクエストを完了扱いにする"""
+    quest = _quest_query(db).filter(models.Quest.id == quest_id).first()
+    if quest is None:
+        raise ValueError(f"Quest {quest_id} not found")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)